modules/migration-combining-must-gather.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="migration-combining-must-gather_{context}"]
+= Combining options when using the must-gather tool
+
+Currently, it is not possible to combine many gather scripts, for example specifying a timeout threshold while permitting insecure TLS connections. Nonetheless, it is feasible to circumvent this restriction in certain instances by establishing internal variables on the must-gather command line, such as:
+
+[source,bash]
```

Comment:
Per our [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output) I think we need to use terminal for oc commands:
```suggestion
[source,terminal]
```